### PR TITLE
Fixed comment in "Feta fransyskor"

### PR DESCRIPTION
**Pull request by StelFoog**
Updated comment to `A` and `B` instead of `D` and `H` to match what is written in the lyrics. Also added `traditionsenligt` to make the comment more of a suggestion (as it tends to be seen).

### CODE REVIEW

File: sangbok/main.xml [hunk 1]
```diff
@@ -1720,7 +1720,7 @@
 		  category="Wihn"
 		  melody="Marche Militaire">
 		<comment>
-			D sjunges av damer; H av herrar
+			A sjunges traditionsenligt av damer; B av herrar
```
Comment (loffa) [inline sangbok/main.xml:1723]:
```suggestion
			Enligt tradition sjunges A av damer; B av herrar
```

Comment (StelFoog) [reply sangbok/main.xml:1723]: I feel like that makes singing only one part optional, but it's probably just how I read it? Apart from that I prefer this suggestion and will change to it if no one else has the same problem when reading it as I do.

Comment (loffa) [reply sangbok/main.xml:1723]: I was thinking the opposite. That only Damer sings traditionally and then Herrar with the original form.

Comment (StelFoog) [reply sangbok/main.xml:1723]: Reading it today I agree with you. I'll commit it!

**Comment by StelFoog** [comment]
@wsv-accidis, is there a problem with the PR?

**Comment by wsv-accidis** [comment]
@StelFoog No problem at all. I just missed the ping, is all. Feel free to poke me!

**Pull request merged**